src/app/views/query-runner/query-input/auto-complete/auto-complete.util.ts

```diff
@@ -51,6 +51,9 @@ function getErrorMessage(queryUrl: string) {
   if (error) {
     return `${translateMessage('Possible error found in URL near')}: ${error}`;
   }
+  if (queryUrl.indexOf('https://graph.microsoft.com/') === -1){
+    return translateMessage('The URL must start with https://graph.microsoft.com/');
```

Comment:
This error isn't appropriate because canary endpoint users use a modified URL that doesn't necessarily look like this

Reply:
Yes. That's right. We should only validate the protocol